docs/data/date-pickers-component-api-pages.ts

```diff
@@ -61,6 +61,7 @@ export default [
     title: 'MultiSectionDigitalClock',
   },
   { pathname: '/x/api/date-pickers/pickers-action-bar', title: 'PickersActionBar' },
+  { pathname: '/x/api/date-pickers/pickers-calendar-header', title: 'PickersCalendarHeader' },
```

Comment:
Would it also make sense adding it into the `Other` section in the `index` page? 🤔